torchvision/datasets/cifar.py

```diff
@@ -31,6 +31,11 @@ class CIFAR10(data.Dataset):
         ['test_batch', '40351d587109b95175f43aff81a1287e'],
     ]
 
+    label_keys = ['labels']
```

Comment:
do we not need `fine_labels` in here too?

Reply:
No, there are no `fine_labels` in CIFAR10, only `labels`, see the old code.